zerver/data_import/import_util.py

```diff
@@ -224,6 +224,34 @@ def build_stream_subscriptions(
 
     return subscriptions
 
+def build_huddle_subscriptions(
+        get_users: Callable[..., Set[int]],
+        zerver_recipient: List[ZerverFieldsT],
+        zerver_huddle: List[ZerverFieldsT]) -> List[ZerverFieldsT]:
```

Comment:
Does it make sense to refactor the Slack import code path to use this function as well?

Reply:
I will create a seperate PR for this.